go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/anchore/syft
+module github.com/dor-hayun/syft
```

Comment:
I don't think this should get changed :)

Reply:
reverted thanks